CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+### 1.76 - 2020-12-01
+
+##### Fixes :wrench:
+
+- Fixed an issue where tileset styles would be reapplied every frame when a tileset has a style and `tileset.preloadWhenHidden` is true. Also fixed a related issue were styles would be reapplied if the style being set is the same as the active style.
```

Comment:
Don't forget to link back to this PR!

Reply:
Whoops, fixed in https://github.com/CesiumGS/cesium/pull/9223/commits/662821bb36e88b57fd41413c01a06c7a8a4aa9d1